test-suite/Makefile

```diff
@@ -176,7 +176,7 @@ summary.log:
 # local build, and downloadable on GitLab)
 report: summary.log
 	$(HIDE)bash save-logs.sh
-	$(HIDE)if [ -n "${TRAVIS}" ]; then find logs/ -name '*.log' -not -name 'summary.log' -exec 'bash' '-c' 'echo "travis_fold:start:coq.logs.$$(echo '{}' | sed s,/,.,g)"' ';' -exec cat '{}' ';' -exec 'bash' '-c' 'echo "travis_fold:end:coq.logs.$$(echo '{}' | sed s,/,.,g)"' ';'; fi
+	$(HIDE)if [ "(" -n "${TRAVIS}" ")" -o "(" -n "${CIRCLECI}" ")" ]; then find logs/ -name '*.log' -not -name 'summary.log' -exec 'bash' '-c' 'echo "travis_fold:start:coq.logs.$$(echo '{}' | sed s,/,.,g)"' ';' -exec cat '{}' ';' -exec 'bash' '-c' 'echo "travis_fold:end:coq.logs.$$(echo '{}' | sed s,/,.,g)"' ';'; fi
```

Comment:
I think you want to add this to the `APPVEYOR` one, or make a separate `CIRCLECI` one, unless `CIRCLECI` also uses `travis_fold:` (which I'm suspicious of).

Reply:
Right, I forgot the reason they're separate.